CalibFormats/SiStripObjects/interface/SiStripClusterizerConditionsGPU.h

```diff
@@ -0,0 +1,139 @@
+#ifndef CalibFormats_SiStripObjects_SiStripClusterizerConditionsGPU_h
+#define CalibFormats_SiStripObjects_SiStripClusterizerConditionsGPU_h
+
+#include "DataFormats/SiStripCluster/interface/SiStripTypes.h"
+
+#include "HeterogeneousCore/CUDACore/interface/ESProduct.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/host_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/HostAllocator.h"
+
+class SiStripQuality;
+class SiStripGain;
+class SiStripNoises;
+
+namespace stripgpu {
+  static constexpr int kStripsPerChannel = 256;
+  static constexpr int kFedFirst = 50;
+  static constexpr int kFedLast = 489;
+  static constexpr int kFedCount = kFedLast - kFedFirst + 1;
+  static constexpr int kChannelCount = 96;
+  static constexpr int kApvCount = 2 * kChannelCount;
+  static constexpr int kStripsPerFed = kChannelCount * kStripsPerChannel;
```

Comment:
in general all of this constants are already available in `DataFormats/SiStripCommon/interface/ConstantsForHardwareSystems.h`